### PR TITLE
[PyOV] Removes unnecessary files from Python wheel

### PR DESCRIPTION
**Pull request by totoka-intel**
### Details:
 - Removes unnecessary copies of the plugins from the whl file

### Tickets:
 - 89839

### CODE REVIEW

File: src/bindings/python/wheel/setup.py [hunk 1]
```diff
@@ -272,14 +274,20 @@ def generate_package(self, src_dirs):
         blacklist = [".lib", ".pdb", "_debug.dll", "_debug.dylib"]
         package_dir = os.path.join(get_package_dir(PY_INSTALL_CFG), WHEEL_LIBS_INSTALL_DIR)
 
+        exclude_patterns = ("*so.*.*.*", "*.so.*.*",
+                            "libpugixml.so", "libtbbmalloc.so", "libtbb.so",
+                            "libopenvino_*.cpython-38-x86_64-linux-gnu.so")
```
Comment (ilya-lavrenov) [inline src/bindings/python/wheel/setup.py:279]: how does it work for other python versions and architectures?

Comment (totoka-intel) [reply src/bindings/python/wheel/setup.py]: Good point. Thanks for suggestion to use sysconfig - hardcoded extension updated, please see the current version.

**Comment by ilya-lavrenov** [comment]
Please, have a look https://github.com/openvinotoolkit/openvino/pull/13725

**Comment by ilya-lavrenov** [comment]
Fixed in another PR.

**Pull request closed without merging**